lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -967,17 +967,23 @@ llvm::json::Value CreateThreadStopped(DAP &dap, lldb::SBThread &thread,
       body.try_emplace("reason", "exception");
       EmplaceSafeString(body, "description", exc_bp->label);
     } else {
+      std::string reason = "breakpoint";
```

Comment:
You do not need to create a `std::string` object, simply use StringRef or char * is enough.

Reply:
updated it